python/cudf/cudf/io/parquet.py

```diff
@@ -716,6 +721,58 @@ def _get_partitioned(
 
 
 class ParquetDatasetWriter:
+    """
+    Write a parquet file or dataset incrementally
+
+    Parameters
+    ----------
+    path : str or s3 and URL
```

Comment:
This annotation doesn't make sense to me. An S3 URL is still a `str`, right?
```suggestion
    path : str
```